search/search-client-rdbms/src/main/java/io/camunda/search/rdbms/RdbmsSearchClient.java

```diff
@@ -126,6 +129,24 @@ public RdbmsSearchClient withSecurityContext(final SecurityContext securityConte
     return this;
   }
 
+  @Override
+  public Long countAssignees(final UsageMetricsQuery query) {
+    throw new UnsupportedOperationException(
+        "UsageMetricsClient countAssignees not implemented yet.");
+  }
+
+  @Override
+  public Long countProcessInstances(final UsageMetricsQuery query) {
+    throw new UnsupportedOperationException(
+        "UsageMetricsClient countProcessInstances not implemented yet.");
+  }
+
+  @Override
+  public Long countDecisionInstances(final UsageMetricsQuery query) {
+    throw new UnsupportedOperationException(
+        "UsageMetricsClient countDecisionInstances not implemented yet.");
+  }
```

Comment:
@ralfpuchert, I believe we should at least add a follow-up issue for this so this is not forgotten. If the RDBMS project has an issue for this already, we need to add a reference to this PR so the RDBMS team is aware.

Reply:
@megglos, are we simply leaving this as is for now? As discussed, this might never become a thing for RDBMS anyway if we want to move this into Zeebe directly. If it does become a requirement, we can still add it to the RDBMS scope later?